core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -519,6 +534,12 @@ void validateManifest(
               entry.sequenceNumber().longValue());
         }
       }
+      if (fileSeqs != null) {
+        V1Assert.assertEquals(
+            "File sequence number should default to 0", (Long) 0L, entry.fileSequenceNumber());
+        V2Assert.assertEquals(
+            "File sequence number should match", fileSeqs.next(), entry.fileSequenceNumber());
+      }
```

Comment:
Nit: would be nice to have empty lines before and after the new block.

Reply:
I agree but it is not how this entire block is formatted. None of the other if statements are separated by empty lines. I decided to follow the existing formatting for consistency even though I'd surround each independent block by empty lines.